workflows/cluster_splitter.nf

```diff
@@ -142,16 +142,16 @@ workflow CLUSTER_SPLITTER {
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
-workflow.onComplete {
-    if (params.email || params.email_on_fail) {
-        NfcoreTemplate.email(workflow, params, summary_params, projectDir, log)
-    }
-    NfcoreTemplate.dump_parameters(workflow, params)
-    NfcoreTemplate.summary(workflow, params, log)
-    if (params.hook_url) {
-        NfcoreTemplate.IM_notification(workflow, params, summary_params, projectDir, log)
-    }
-}
+//workflow.onComplete {
```

Comment:
Could you delete this instead of commenting this out?

Reply:
fixed in: https://github.com/phac-nml/clustersplitter/pull/1/commits/7b234b1a1fc8f330f895cca80afe115f4dff5981